packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -28,7 +28,7 @@ import UpdateBookmarkCommand from './updatebookmarkcommand.js';
 
 import '../theme/bookmark.css';
 
-import bookmarkIcon from '../theme/icons/bookmark_inline.svg';
+const bookmarkIcon = icons.bookmarkInline;
```

Comment:
Use `icons.bookmarkInline` directly instead of creating a global variable, as global variable prevents the entire `icons` object from being tree-shaken (https://github.com/ckeditor/ckeditor5/pull/17345).